lib/drivers/nest-protocol.js

```diff
@@ -0,0 +1,59 @@
+const Driver = require('../models/driver');
+const Supply = require('../models/supply');
+
+/**
+ * NestProtocol driver. Supports circulating supply
+ * for token NEST on Ethereum blockchain.
+ *
+ * @augments Driver
+ * @memberof Driver
+ */
+class NestProtocol extends Driver {
+  constructor(options) {
+    super({
+      timeout: 100, // 10 requests per second
+      supports: {
+        total: false,
+        circulating: true,
+        max: true,
+        blockchains: ['Ethereum'],
```

Comment:
The blockchain is not needed here. It doesn't have support for all the tokens on Ethereum, just Nest protocol
```suggestion
```